homeassistant/components/youtube/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "domain": "youtube",
+  "name": "YouTube",
+  "codeowners": ["@joostlek"],
+  "config_flow": true,
+  "dependencies": ["application_credentials"],
+  "documentation": "https://www.home-assistant.io/integrations/youtube",
+  "integration_type": "service",
+  "iot_class": "cloud_polling",
+  "requirements": ["google-api-python-client==2.71.0"]
```

Comment:
Although still supported, Google recommends other libraries for new code. Is there a better alternative?

Reply:
Do you have a link for that? When implementing I only saw them using this lib

Reply:
https://github.com/googleapis/google-api-python-client/#other-google-api-libraries

Reply:
I don't see Youtube in the list https://github.com/googleapis/google-cloud-python. So I think this lib is only used for Google Cloud and not for All apis yet

Reply:
Ok. 👍